README.md

```diff
@@ -14,20 +14,22 @@ This template can be used to quickly create an [Apollo Federation] subgraph with
 
 ## Next Steps
 
-- Setup project with `npm install` 
-- Download [Rover] and start it using the command printed out from `cargo run` to start a local version of Apollo Explorer.
+- Setup project with `npm install`.
+- Start the server with `npm start` to start a local version of Apollo Explorer.
```

Comment:
I _think_ the `cargo run` command was just a copypasta error but I could be wrong!

Reply:
Great catch!